.gitignore

```diff
@@ -43,9 +43,20 @@ local_test/*
 !local_test/index.html
 local_test/js/*
 local_test/css/*
+# folder of projects to test 
+local_test/intesting
 
 # local test code
 localTestCode.txt
 
 # Apple files
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+
+# textmate
+code_textmate
+code_textmate.tmproj
+
+package-lock.json
```

Comment:
This file was included by @tbushman. I'm using version 3.5.2 of npm. @tbushman maybe you should consider using an older version of npm for this project so the lock file isn't produced?